### PR TITLE
Athena fix

### PR DESCRIPTION
**Pull request by ciebson123**
# Description
Added dataset to athena, added default paths, less name fuckery

# Checklist
   - [x] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [x] Are all variables set with logical defaults that are backward compatible?
   - [x] I attached link to neptune if it was necessary
   - [x] Have you successfully executed the linter and tests?
   - [x] Are all functions concise and don't require splitting?
   - [x] Is there documentation for arguments and complex logic?
   - [x] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [x] Are all functions placed in appropriate files?
   - [x] Is the PR name meaningful and descriptive?
   - [x] Is PR description provided, or unnecessary?
  
Other questions:
   - [x] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [x] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [x] If there is a need for a second reviewer I requested additional review 

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

File: lizrd/scripts/download_c4/download_c4.sh
Comment (crewtool) [inline lizrd/scripts/download_c4/download_c4.sh:1]: gpu is unnecessary when downloading dataset

File: lizrd/scripts/download_c4/README.md
Comment (crewtool) [inline lizrd/scripts/download_c4/README.md:1]: Isn't it easier to just clone the repo and  run ./start-dev?

File: lizrd/support/sync_code.py [hunk 1]
```diff
@@ -80,14 +80,15 @@ def athena_user_to_workdir(connection):
         "plgmaciejpioro": "maciejpioro",
         "plgludziej": "ludziej",
         "plgsimontwice": "simontwice",
+        "plgkciebiera": "plgkciebiera",
```
Comment (crewtool) [inline lizrd/support/sync_code.py:83]: athena_user_to_workdir is not used anymore

File: submit_experiment.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def submit_experiment(
     with ConnectWithPassphrase(hostname) as connection:
         result = connection.run("uname -n", hide=True)
         node = result.stdout.strip()
-        cluster = get_machine_backend(node)
+        cluster = get_machine_backend(node, connection)
```
Comment (crewtool) [inline submit_experiment.py:60]: connection is passed only for accesing the username, so i see two options:
1. similarly to `node` we get the username before
2. We pass connection and extract node name inside `get_machine_backend`

**Review by crewtool — CHANGES_REQUESTED** [review]
left some comments

File: lizrd/scripts/download_c4/README.md
Comment (ciebson123) [reply lizrd/scripts/download_c4/README.md:1]: I think we wanted minimal requirements possible.

File: submit_experiment.py (hunk 2 above)
Comment (ciebson123) [reply submit_experiment.py:60]: Option 2 is better imo, implemented

File: lizrd/support/sync_code.py (hunk 1 above)
Comment (ciebson123) [reply lizrd/support/sync_code.py]: Yeeted

File: lizrd/scripts/download_c4/download_c4.sh
Comment (ciebson123) [reply lizrd/scripts/download_c4/download_c4.sh:1]: I think this is now done?

**Review by crewtool — APPROVED** [review]
LGTM

**Pull request merged**